python/cudf_polars/tests/expressions/test_stringfunction.py

```diff
@@ -150,3 +150,51 @@ def test_slice_column(slice_column_data):
     else:
         query = slice_column_data.select(pl.col("a").str.slice(pl.col("start")))
     assert_ir_translation_raises(query, NotImplementedError)
+
+
+@pytest.mark.parametrize(
+    "target, repl",
+    [("a", "a"), ("Wı", "☺"), ("doesnotexist", "blahblah")],  # noqa: RUF001
```

Comment:
Can we test with the replacement being an empty string please? Same with replace_many